### PR TITLE
Ubuntu intro README 

### PR DESCRIPTION
**Pull request by Karyum**
The structure of the README is WHAT?WHY?HOW?
in the how i was thinking of having the actual Ubuntu setup on the student's laptop.
and straight after that we have the cli-intro workshop.

is this structure fine for everyone ?

### CODE REVIEW

File: week1/ubuntu-intro.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+> **open-source** meaning that everone can view the code of Ubuntu, and anyone can contribute to it's progress.
+
+## Why ?
+
+- It's lightweight and high performant, unlike windows.
+- It's free, and it also comes with a massive library of tools that are also free to download.
+- Better security and privacy
+> Linux ensures the privacy of user’s data as it never collects much data from the user while using its distributions or software but this is not true for many other operating systems.
+- Gives you more power over your computer since it's a lower layer  <!-- help needed -->
```
Comment (Karyum) [inline week1/ubuntu-intro.md]: i also i need help phrasing this correctly i'm not sure how.

**Comment by Karyum** [comment]
~also thinking about it maybe i should add like a "Further reading" section since we mentioned we wanted like a home work system for the precourse~

**Comment by Karyum** [comment]
~TODO: add `Further reading` and `Advanced topics`~

File: week1/ubuntu-intro.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- And then shrink the volume (hard disk) to 20 - 40 GB.
+
+![image alt](https://i.imgur.com/L0daELC.gif)
+
+### Step 3
+
+- Restart your computer and enter the BIOS Setup. **Make sure to have the usb stick in the computer**
+     - On every computer the entry to BIOS key is different, so you should go on Google and search how to get in the BIOS for you paticular computer's model. or here's a list for the common BIOS keys for different manufacturers:
+  
+    - Acer: F2 or DEL
```
Comment (shiryz) [inline week1/ubuntu-intro.md]: All of these appear on the same line

**Review by shiryz — CHANGES_REQUESTED** [review]

File: week1/ubuntu-intro.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+![image alt](https://i.imgur.com/OUhvP5T.gif)
+
+### Step 2
+
+- Choose which disk you want to partition.
+> Partitioning is the process of dividing a disk into one or more regions, the so called partitions. partitioning a hard disk is helpful for either better organization or for having 2 operating systems on the same hard disk. 
+
+- And then shrink the volume (hard disk) to 20 - 40 GB.
+
```
Comment (jamiecoe) [inline week1/ubuntu-intro.md:47]: I'd suggest that the students review that they've done Step 2 correctly with a mentor before continuing. If they get this step wrong, they could potentially corrupt their hard-drive.

**Review by jamiecoe — CHANGES_REQUESTED** [review]

**Comment by Karyum** [comment]
@shiryz @jamiecoe changes accoring to https://github.com/FACN8/Pre-course/pull/1#pullrequestreview-321963890 https://github.com/FACN8/Pre-course/pull/1#pullrequestreview-321963641 are done

**Review by shiryz — APPROVED** [review]

**Comment by shiryz** [comment]
@Karyum merge conflicts

**Review by jamiecoe — APPROVED** [review]

**Pull request merged**